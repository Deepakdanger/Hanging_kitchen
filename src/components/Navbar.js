import React from 'react';

const Navbar = () => (
  <nav className="nav1">
    <div className="nav_title1">
      <div className="nav_title2">
        <div className="title">
          <h1 className="pro">Han</h1>
          <h1 className="glow">gin</h1>
          <h1 className="pro">g</h1>
        </div>
        <div>
          <h1 className="pro1 rotated">Kitchen</h1>
        </div>
      </div>
    </div>
    <p className="right">
      <p className="data">Home</p>
      <p className="data">Search</p>
    </p>
  </nav>
);

export default Navbar;
