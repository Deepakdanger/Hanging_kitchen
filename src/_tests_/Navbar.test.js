import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import { StaticRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Testing Nav with router', () => {
  it('Nav is defined', async () => {
    const component = render(
      <StaticRouter>
        <Navbar />
      </StaticRouter>,
    );
    expect(component).toBeDefined();
  });

  it('Nav snapshot testing', () => {
    const nav = renderer.create(
      <StaticRouter>
        <Navbar />
      </StaticRouter>,
    ).toJSON();
    expect(nav).toMatchSnapshot();
  });
});
