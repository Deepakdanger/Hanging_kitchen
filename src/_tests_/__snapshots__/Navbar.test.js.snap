// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Nav with router Nav snapshot testing 1`] = `
<nav
  className="nav1"
>
  <div
    className="nav_title1"
  >
    <div
      className="nav_title2"
    >
      <div
        className="title swing1"
      >
        <h1
          className="pro"
        >
          Han
        </h1>
        <h1
          className="glow"
        >
          gin
        </h1>
        <h1
          className="pro"
        >
          g
        </h1>
      </div>
      <div>
        <h1
          className="pro1"
        >
          Kitchen
        </h1>
      </div>
    </div>
  </div>
  <p
    className="right"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <span
        className="data"
      >
        Home
      </span>
    </a>
  </p>
</nav>
`;

exports[`Testing Nav with router Navbar snapshot testing 1`] = `
<nav
  className="nav1"
>
  <div
    className="nav_title1"
  >
    <div
      className="nav_title2"
    >
      <div
        className="title swing1"
      >
        <h1
          className="pro"
        >
          Han
        </h1>
        <h1
          className="glow"
        >
          gin
        </h1>
        <h1
          className="pro"
        >
          g
        </h1>
      </div>
      <div>
        <h1
          className="pro1"
        >
          Kitchen
        </h1>
      </div>
    </div>
  </div>
  <p
    className="right"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <span
        className="data"
      >
        Home
      </span>
    </a>
  </p>
</nav>
`;
